webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/CardRegistry.js

```diff
@@ -2,10 +2,16 @@ import React from 'react';
 import { addGlobalFill } from '../../../common/Fill/GlobalFill';
 import Properties from '../Details/Cards/SystemProperties';
 import OperatingSystem from '../Details/Cards/OperatingSystem';
+import TemplatesCard from '../Details/Cards/TemplatesCard';
 
 const cards = [
   { key: '[core]-properties-card', Component: Properties, weight: 4000 },
   { key: '[core]-os-card', Component: OperatingSystem, weight: 3000 },
+  {
+    key: '[core]-templates-card',
+    Component: TemplatesCard,
+    weight: 2000,
```

Comment:
The card takes up a lot of vertical space (when there are 8 templates) and makes the overall layout look kind of strange since the smaller cards are pushed to the right:

![templates-card](https://user-images.githubusercontent.com/22042343/191835194-935be661-1dae-49ef-aaab-2140026ec482.png)

What do you think about decreasing the weight so the card appears last? I think it would look better that way.